conf4j-javassist/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>conf4j-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
The dependency to compatibility kit must be added with **test** scope - now this module has dependencies to junit and other test modules. 

conf4j has convention to put the test dependencies in the section `<!-- Test only dependencies -->` - here is the change you should apply (both in conf4j-javassist/pom.xml and conf4j-spring/pom.xml):

```
        <!-- Test only dependencies -->
        <dependency>
            <groupId>com.sabre.oss.conf4j</groupId>
            <artifactId>conf4j-ck</artifactId>
            <version>${project.version}</version>
            <scope>test</scope>
        </dependency>
```